knowledge-base/how-to-set-the-report-property-of-the-webreportdesigner-dynamically-through-model.md

```diff
@@ -0,0 +1,106 @@
+---
+title: How to set the Report Property of the WebReportDesigner Dynamically Through Model
+description: Explains how to select the report that will be edited in the web report designer from a view or query it in the URL
+type: how-to
+page_title: Setting the Report Property Dynamically in the WebReportDesginer in .NET Core MVC
```

Comment:
Consider changing to "ASP.NET MVC and ASP.NET Core MVC."
Should work also in ASP.NET MVC